app/docker/networks/edit/NetworkDetailsView.tsx

```diff
@@ -0,0 +1,96 @@
+import { useRouter, useCurrentStateAndParams } from '@uirouter/react';
+import DockerNetworkHelper from 'Docker/helpers/networkHelper';
+
+import { useEnvironmentId } from '@/portainer/hooks/useEnvironmentId';
+import { PageHeader } from '@/portainer/components/PageHeader';
+import { confirmDeletionAsync } from '@/portainer/services/modal.service/confirm';
+import * as notifications from '@/portainer/services/notifications';
+// import { AccessControlPanel } from '@/portainer/access-control/AccessControlPanel/AccessControlPanel';
+
+import { removeNetwork, isSystemNetwork } from '../network.service';
+import { useNetwork } from '../queries';
+import { IPConfigs } from '../types';
+
+import { NetworkDetailsTable } from './NetworkDetailsTable';
+
+export function NetworkDetailsView() {
+  const router = useRouter();
+
+  const {
+    params: { id: networkId }, // nodeName
+  } = useCurrentStateAndParams();
+  const environmentId = useEnvironmentId();
+
+  const networkQuery = useNetwork(networkId, environmentId); // 1 object
+
+  const allowRemoveNetwork: boolean =
+    networkQuery.data && !isSystemNetwork(networkQuery.data.Name);
+  const IPV4Configs: IPConfigs =
+    networkQuery.data?.IPAM &&
+    DockerNetworkHelper.getIPV4Configs(networkQuery.data?.IPAM.Config);
+  const IPV6Configs: IPConfigs =
+    networkQuery.data?.IPAM &&
+    DockerNetworkHelper.getIPV6Configs(networkQuery.data?.IPAM.Config);
+
+  // if there's a network error, tell the user
+  if (networkQuery.isError) {
```

Comment:
this will send an error on every rerender. my suggestion is to put it in `useQuery` options. so:
`useQuery(key, () => getNetwork(), { onError(error) { notifyError(...) } }```

Reply:
updated